internal/definitions/sender_tokenpool_test.go

```diff
@@ -56,10 +56,10 @@ func TestBroadcastTokenPoolInvalid(t *testing.T) {
 	mdm.AssertExpectations(t)
 }
 
-func TestBroadcastTokenPoolInvalidNonMultiparty(t *testing.T) {
+func TestBroadcastTokenPoolInvalidMultiparty(t *testing.T) {
```

Comment:
Is this test now identical to the previous one?

Reply:
heh not quite - wasn't working in gateway mode bc the error checking changed. So I'll actually fix this test as part of adding new unit / E2E tests.